### ISSUE TITLE
Using rel="manifest"

### ISSUE THREAD

**Issue by HadrienGardeur** [w3c/wpub#132]
I'd like to re-visit our decision to roll our own rel value (`publication`) for detecting the WP manifest.

As I've said in the past, I don't think there's any good reason why we can't use `manifest` instead:

- if we adopt the WAM for our serialization, we'll need to use it anyway
- even if we don't adopt the WAM, the definition for `manifest` is a perfectly fit for our use case (https://w3c.github.io/manifest/#link-relation-type-registration) and we can easily identify the WP manifest using a media type

We could also immediately get rid of the first section of the lifecycle as well, and simply reference the WAM section instead.

I really think that this is a straightforward decision, and an easy win (less spec language and monkey patching in our draft).

**Comment by rdeltour** [comment]
If we adopt WAM, I definitely agree (and hadn’t actually realized it was an agreed decision).

If we don’t however, or if we end up using a complementary manifest format for things like reading order, wouldn’t it be preferable to use a different value to avoid conflicting with WAM?

**Comment by iherman** [comment]
What @rdeltour said, and something more. The WAM specification, as well as our own lifecycle definition, says that the User Agent must pick the _first_ reference to a manifest. If we do not use the WAM, and the user puts two manifest references into the starting page, one for a WAM and the other for the WPUB Manifest, then one will loose.

I know we were considering using a `profile` in the media type to differentiate between the two. However, I am not sure all user agents rely on the media type (ie, whether they look at it at all...) and, if not, we create a potential problem.

I would propose to postpone this issue until we make a final decision on the WAM or not WAM...

**Comment by HadrienGardeur** [comment]
No, it's not preferable to have a distinct rel value.

A rel is not tied to a given media type, otherwise, why would we have rel values in the first place? We could simply use the media type.

It's true that the WAM processing would look for the first link though, this would require to filter based on media type as well if we adopt a different serialization. This would affect how we obtain the manifest slightly, but it's not a valid reason for defining a new rel value.

**Comment by HadrienGardeur** [comment]
By the way, IANA link registry also contains `collection`. It's another option available to us (a Web Publication is by definition a collection).

**Comment by rdeltour** [comment]
> By the way, IANA link registry also contains collection. It's another option available to us (a Web Publication is by definition a collection).

Yes, which was also my point when saying that another value _might_ be preferable. It also depends on how the `manifest` relation has been defined (I didn’t find the registration request on the [list-relations mailing list](https://www.ietf.org/mailman/listinfo/link-relations) btw).

As @iherman said, let’s define what we want to link first, then only figure out the proper relation.

**Comment by HadrienGardeur** [comment]
The definition is in the draft:

> Links to a manifest. A manifest provides developers with a centralized place to put metadata associated with a web application.

IMO this is not quite generic enough for an official registration, it would probably get rejected as-is.
